userland/libtock/led.c

```diff
@@ -1,18 +1,20 @@
-#include "led.h"
-#include "tock.h"
+#include "internal/led.h"
+
+// For LEDs, these simply pass through to the syscall wrappers (the functions
+// in the internal folder).
```

Comment:
I also agree that I don't understand why exactly. @bradjc how do you feel about us merging this back into the non-internal files?